test/features/deploy_source_files.feature

```diff
@@ -0,0 +1,50 @@
+@smoke
```

Comment:
Please add a `@cli` tag, these tests can be executed just with CLI.
Also, should they be marked as smoke?

Reply:
Will add the @cli, about @smoke, maybe not as is not considered basic functionality, right? @radtriste

Reply:
I would not put the whole feature as smoke. Just one like the `Scenario: Deploy .dmn source files with CLI`

Reply:
Done